mods/ra/maps/soviet-soldier-volkov-n-chitzkoi/map.yaml

```diff
@@ -0,0 +1,1449 @@
+MapFormat: 11
+
+RequiresMod: ra
+
+Title: Soviet Soldier Volkov & Chitzkoi
+
+Author: Westwood Studios
+
+Tileset: SNOW
+
+MapSize: 128,128
+
+Bounds: 18,23,60,73
+
+Visibility: MissionSelector
+
+Categories: Mission
+
+LockPreview: True
+
+Players:
+	PlayerReference@Neutral:
+		Name: Neutral
+		OwnsWorld: True
+		NonCombatant: True
+		Faction: england
+	PlayerReference@Creeps:
+		Name: Creeps
+		NonCombatant: True
+		Faction: england
+        Enemies: Greece, GoodGuy, Spain, France, USSR
```

Comment:
For the tooltip description i have two questions:
1) How do you make it show ingame? Can't find the setting.
2) Do you know wich trait point to it? Because I'm having a hard time finding it.
Thx!

Reply:
The tooltip should just appear when you hover with your mouse over the building. The trait is `TooltipDescription` iirc.